scripts/__init__.py

```diff
@@ -0,0 +1,6 @@
+#from . import modules
```

Comment:
should we remove these commented-out imports completely? I suppose they are left-overs from before you re-shuffled the structure

Reply:
yep, those are leftovers, I've removed them